src/browserview/BrowserComponent.java

```diff
@@ -252,9 +252,11 @@ public void jumpToComment(){
 	
 	private boolean isBrowserActive(){
 		if (driver.getCurrentUrl().isEmpty() && driver != null){
+			logger.warn("Unable to read url from bview. Resetting.");
 			return false;
 		}
 		else if (driver == null){
+			logger.warn("chromedriver process was killed !");
 			return false;
 		}
```

Comment:
hmm it is possible that it throws this exception. shall impement a try catch block here